builtin/providers/aws/provider.go

```diff
@@ -190,6 +190,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_cloudwatch_log_group":                     resourceAwsCloudWatchLogGroup(),
 			"aws_cloudwatch_log_metric_filter":             resourceAwsCloudWatchLogMetricFilter(),
 			"aws_cloudwatch_log_subscription_filter":       resourceAwsCloudwatchLogSubscriptionFilter(),
+			"aws_config_configuration_recorder":            resourceAwsConfigConfigurationRecorder(),
```

Comment:
line is missing:

"aws_config_delivery_channel":                  resourceAwsConfigDeliveryChannel(),